inference-engine/src/mkldnn_plugin/ngraph_transformations/op/swish_cpu.hpp

```diff
@@ -11,6 +11,7 @@ namespace MKLDNNPlugin {
 class SwishNode : public ngraph::op::Op {
 public:
     static constexpr ngraph::NodeTypeInfo type_info{"SwishCPU", 0};
+    static constexpr const ::ngraph::Node::type_info_t& get_type_info_static() { return type_info; }
```

Comment:
Why don't you use `NGRAPH_RTTI_DECLARATION` macro?

Reply:
I can't use NGRAPH_RTTI_DECLARATION macro because this macro defines static functions which should be a part of public API. Also users should find this function with dlsym before using.